src/CommonDBTM.php

```diff
@@ -3708,7 +3708,11 @@ public function getNameID($options = [])
 
         if (
             $p['forceid']
-            || $_SESSION['glpiis_ids_visible']
+            ||
+            (
+                isset($_SESSION['glpiis_ids_visible'])
```

Comment:
This change does not seems strictly related to current PR

Reply:
yes, it's, this PR (from update step) use function to inject XML files containing rules.
Without, we have a warning

Reply:
ok